pkg/server/router.go

```diff
@@ -53,6 +56,14 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 	m.PathPrefix("/dashboard/").Handler(vueUI.IndexFileOnNotFound())
 	m.PathPrefix("/api-ui").Handler(vueUI.ServeAsset())
 
+	sbDownloadHandler := supportbundle.NewDownloadHandler(r.server.Context, r.server.namespace, r.scaled)
+	downloadRoute := m.Path("/v1/supportbundles/{bundleName}/download").Methods("GET")
```

Comment:
Check if we can update schema links instead of changing the routes here
Example:
https://github.com/harvester/harvester/blob/64e4d950ec731040495567b7485175df813339d4/pkg/api/vmtemplate/formatter.go#L8
https://github.com/harvester/harvester/blob/64e4d950ec731040495567b7485175df813339d4/pkg/api/vmtemplate/template_handler.go#L22

Reply:
working on this.

Reply:
What's the conclusion for this?

Reply:
To add a link we need to create another resource and store for it, I would suggest not to do it if the link `download` simply contains a string `/v1/supportbundles/<bundlename>/download`.